### PR TITLE
[RW-7604][risk=no] Emphasize cost warning on genomic extraction modal

### PR DESCRIPTION
**Pull request by calbach**
image

### CODE REVIEW

**Review by NehaBroad — APPROVED** [review]

**Comment by calbach** [comment]
Per workbench-ux discussion, added a support link here. See updated screenshot. Feel free to review if you see this, but probably won't wait on approval as it is a small change.

**Pull request merged**
